v23.1/take-full-and-incremental-backups.md

```diff
@@ -149,11 +149,19 @@ To view the available backup subdirectories, use [`SHOW BACKUPS`](show-backup.ht
 To take incremental backups, you need an [Enterprise license](enterprise-licensing.html).
 {{site.data.alerts.end}}
 
-If your cluster grows too large for nightly [full backups](#full-backups), you can take less frequent full backups (e.g., weekly) with nightly incremental backups. Incremental backups are storage efficient and faster than full backups for larger clusters.
+Incremental backups are chains between full backups and contain only the data that has changed since a base set of backups (which must include one full backup, and can include many incremental backups). Incremental backups are smaller and faster to produce than full backups. You can take incremental backups either as of a given timestamp or with full [revision history](take-backups-with-revision-history-and-restore-from-a-point-in-time.html).
```

Comment:
Line 152-160 --> I recommend moving to the end of this section.

Reply:
@livlobo did you mean moving to the end of the incremental section or still above [Garbage Collection](https://www.cockroachlabs.com/docs/stable/take-full-and-incremental-backups.html#take-an-incremental-backup)?